src/site/stages/build/options.js

```diff
@@ -97,9 +97,13 @@ function applyDefaultOptions(options) {
       destination: './',
     },
     destination: path.resolve(projectRoot, 'build', options.buildtype),
-    appAssets: {
-      source: '../../assets',
-      destination: './',
+    fontAssets: {
+      source: '../../assets/fonts',
+      destination: './fonts',
+    },
+    imageAssets: {
+      source: '../../assets/img',
+      destination: './img',
```

Comment:
By splitting up `/assets`, we can have webpack be responsible for `assets/js` and let metalsmith take care of the fonts and images.